src/hotspot/share/cds/cds_globals.hpp

```diff
@@ -127,6 +127,27 @@
   product(bool, AOTCacheParallelRelocation, true, DIAGNOSTIC,               \
           "Use parallel relocation code to speed up startup.")              \
                                                                             \
+  /* AOT Code flags */                                                      \
+                                                                            \
+  product(bool, AOTCodeCaching, false, DIAGNOSTIC,                          \
```

Comment:
This PR is not using `AOTCodeCaching` flag. Does it make sense to add it later when support for nmethods is added?

Reply:
Yes.

Originally I used it as main flag to disable all code caching including stubs and adapters. But now it will only control JITed AOT code.